resemble.js

```diff
@@ -478,6 +478,16 @@ var _this = {};
 					if(hasMethod) { param(); }
 					return self;
 				},
+				setTolerance: function(value){
+
+					tolerance.red = value;
+					tolerance.green = value;
+					tolerance.blue = value;
+					tolerance.alpha = value;
+
```

Comment:
Good point.  Following the pattern of other APIs, I changed setTolerance to reset ignoreColors, otherwise it would have no effect if called after ignoreColors().  It should not reset ignoreAntialiasing or brightness settings, so that it can potentially be used in conjunction with either ignoreNothing (the default) or ignoreAntialiasing.